src/components/InputForm/InputForm.js

```diff
@@ -1,19 +1,23 @@
-import {useNote} from "../../context/Note-Context"
-import { BsFillPinFill,BsTrash } from "react-icons/bs";
+import {v4 as uuid} from "uuid"
+import {useNote,useArchive} from "../../context"
+
+import { BsFillPinFill } from "react-icons/bs";
 import { IoIosColorPalette , IoIosAddCircleOutline} from "react-icons/io";
 import { BiArchiveIn } from "react-icons/bi";
+
 import "./inputForm.css"
 
 
```

Comment:
remove spacing

Reply:
Space is given only for readability purpose